Plugins/ServerPlugin.cpp

```diff
@@ -240,11 +240,6 @@ void CompilerClient::UpdateLoop() {
 }
 
 ServerPlugin::ServerPlugin(MainWindow& mainWindow) : RGMPlugin(mainWindow) {
-  /* 

```

Comment:
You don't seem to be checking the new macros you created. This code's just always uncommented. Are you just relying on it not being called?

Reply:
I am checking the define `RGM_SERVER_ENABLED` in `MainWindow.cpp` which calls this constructor. The plugin interface I designed was meant to be pluggable into RGM, so fundies kind of went commenting it out in the wrong place.